### PR TITLE
Fix issue where non-applied filters are shown if another that is applied exists

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder?_sp.spa=pr-filter-summary-exclude-unapplied](https://developer.blackbaud.com/skyux-list-builder?_sp.spa=pr-filter-summary-exclude-unapplied)

File: src/app/public/modules/list-filters/list-filter-summary.component.spec.ts [hunk 1]
```diff
@@ -112,6 +112,60 @@ describe('List filter summary', () => {
     });
   }));
 
+  it('should show filter summary in the appropriate area when filter a filters which have no effect exists', async(() => {
```
Comment (blackbaud-svossen) [inline src/app/public/modules/list-filters/list-filter-summary.component.spec.ts:115]: might wanna fix the way this reads; `filter a filters`

**Review by blackbaud-svossen — DISMISSED** [review]
Approved whether or not you take the suggestion. Let me know if you need me to mark it approved again.

File: src/app/public/modules/list-filters/list-filter-summary.component.spec.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/list-filters/list-filter-summary.component.spec.ts:115]: I think it actually just needs to be `when filters which have no effect exist`. The `filter a` is copy pasta I think.

**Review by blackbaud-svossen — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder?_sp.spa=pr-filter-summary-exclude-unapplied](https://developer.blackbaud.com/skyux-list-builder?_sp.spa=pr-filter-summary-exclude-unapplied)

**Pull request merged**